tests/sentry/web/frontend/test_react_page.py

```diff
@@ -132,3 +132,65 @@ def extract_url_names(urlpatterns, parents):
 
                 assert response.status_code == 302
                 assert response["Location"] == f"http://testserver{path}"
+
+    def test_redirect_to_customer_domain(self):
+        user = self.create_user("bar@example.com")
+        org = self.create_organization(owner=user)
+
+        self.login_as(user)
+
+        with self.feature({"organizations:customer-domains": False}):
+            assert "activeorg" not in self.client.session
+
+            response = self.client.get(reverse("sentry-organization-issue-list", args=[org.slug]))
+            assert response.status_code == 200
+            assert self.client.session["activeorg"]
+
+        with self.feature({"organizations:customer-domains": True}):
+
+            # Redirect to customer domain
+            response = self.client.get(
+                reverse("sentry-organization-issue-list", args=[org.slug]), follow=True
+            )
+            assert response.status_code == 200
+            assert response.redirect_chain == [
+                (f"http://{org.slug}.testserver/organizations/{org.slug}/issues/", 302)
```

Comment:
Won't this result in another redirect as the request has to end up on a path without the organization slug? Perhaps we can follow this change up with another one that refines how redirects are done 🤷

Reply:
This is intentional for this test case. We currently don't have redirects on the Django side to redirect to paths with the organization slug omitted for when a customer domain is accessed.

The redirect you're thinking of would occur on the frontend app.